.github/workflows/lighthouse.yml

```
@@ -0,0 +1,96 @@
+          echo "token=$TOKEN" >> $GITHUB_OUTPUT
+
+      - name: Audit preview URL with Lighthouse
+        id: lighthouse_audit
+        uses: treosh/lighthouse-ci-action@v11
+        with:
+          urls: |
+            http://localhost/dashboard
+            http://localhost/dashboard/map
+          uploadArtifacts: true
+          temporaryPublicStorage: true
+          extraHeaders: '{"Cookie": "token=${{ steps.login.outputs.token }}"}'
```

actionlint (1.7.7)

input "extraHeaders" is not defined in action "treosh/lighthouse-ci-action@v11". available inputs are "artifactName", "basicAuthPassword", "basicAuthUsername", "budgetPath", "configPath", "runs", "serverBaseUrl", "serverToken", "temporaryPublicStorage", "uploadArtifacts", "uploadExtraArgs", "urls"

(action)